api/proto/teleport/legacy/types/types.proto

```diff
@@ -3439,6 +3439,15 @@ message RoleConditions {
     (gogoproto.nullable) = false,
     (gogoproto.jsontag) = "github_permissions,omitempty"
   ];
+
+  // WorkloadIdentityLabels controls whether or not specific WorkloadIdentity
+  // resources can be invoked. Further authorization controls exist on the
+  // WorkloadIdentity resource itself.
+  wrappers.LabelValues WorkloadIdentityLabels = 44 [
```

Comment:
Should we also add role expressions? All resources have them

Reply:
+1 for consistency, it's terribly frustrating when we support expressions on every field but the one I try

Reply:
👍  Sure - easy enough to do. I wasn't sure how widespread their use was.

Reply:
[65ed124](https://github.com/gravitational/teleport/pull/50071/commits/65ed124f872f33767db2a2e1e6ba29c9e7fff478)